app/src/main/java/com/ataulm/whatsnext/account/IsSignedInUseCase.kt

```diff
@@ -0,0 +1,10 @@
+package com.ataulm.whatsnext.account
+
+import com.ataulm.whatsnext.TokensStore
+
+internal class IsSignedInUseCase(private val tokensStore: TokensStore) {
```

Comment:
It's a bit boilerplate-y to have this admittedly unnecessary level of indirection, but for me, it's preferable to have in the ViewModel rather than the TokensStore directly -> using high level abstractions in the ViewModel makes it easier to understand what the role of the collaborators are.